src/libraries/System.Diagnostics.Process/src/Microsoft/Win32/SafeHandles/SafeProcessHandle.cs

```diff
@@ -19,6 +19,9 @@ public sealed partial class SafeProcessHandle : SafeHandleZeroOrMinusOneIsInvali
     {
         internal static readonly SafeProcessHandle InvalidHandle = new SafeProcessHandle();
 
+        /// <summary>
+        /// Creates a SafeProcessHandle
```

Comment:
```suggestion
        /// Creates a SafeProcessHandle.
```

Reply:
Also, should the `SafeProcessHandle` be `<see cref="SafeProcessHandle"/>`?  Same question for other occurrences.